Makefile

```diff
@@ -63,9 +69,10 @@ build-image:
 		virtualenv ~/concreate; \
 		source ~/concreate/bin/activate; \
 		pip install -U concreate==1.3.0; \
-		concreate generate --target target-docker; \
+		$(CONCREATE_CMD) \
 		deactivate; \
 	)
+	@echo "Building image " $(DEV_IMAGE_FULL_NAME)
```

Comment:
Make already prints out commands as it goes... so you will see the full name printed in the line below. Would that be sufficient or you still prefer to have a separate echo?